straxen/plugins/events/event_positions.py

```diff
@@ -116,34 +116,21 @@ def compute(self, events):
         for s_i in [0, 1, 2]:
             # alt_sx_interaction_drift_time is calculated between main Sy and alternative Sx
             drift_time = events['drift_time'] if not s_i else events[f'alt_s{s_i}_interaction_drift_time']
-
             z_obs = - self.electron_drift_velocity * drift_time
             xy_pos = 's2_' if s_i != 2 else 'alt_s2_'
             orig_pos = np.vstack([events[f'{xy_pos}x'], events[f'{xy_pos}y'], z_obs]).T
             r_obs = np.linalg.norm(orig_pos[:, :2], axis=1)
-            delta_r = self.map(orig_pos)
             z_obs = z_obs + self.electron_drift_velocity * self.electron_drift_time_gate
 
-            # apply radial correction
+            # apply z bias correction
+            z_dv_delta = self.z_bias_map(np.array([r_obs, z_obs]).T, map_name='z_bias_map')
+            corr_pos = np.vstack([events[f"{xy_pos}x"], events[f"{xy_pos}y"], z_obs - z_dv_delta]).T
+            # apply r bias correction
+            delta_r = self.map(corr_pos)
```

Comment:
@shenyangshi If I am correct, you changed the coordinate the map applied on from `z_obs ` to `z_obs - z_dv_delta`. So actually you will see a difference between results before and after this PR. So I do not understand why the difference is so small.

Reply:
Right, because the Z correction is only happening at the bottom outside corner, so other events won't be changed by a lot.

Reply:
So we actually do not expect a non-zero difference, no? And whatever the master branch gives us, it is wrong.

Reply:
No we do expect a non-zero difference for R, as the z_bias map will do some nearly negligible thing inside the bulk tpc too. What I don't understand is why the z has a change, but it's also negligible.